wiki/Contests/pl.md

```diff
@@ -45,7 +45,7 @@ Konkursy Aspire to konkursy dot. tworzenia beatmap, w których nie obowiązują
 
 Ważne:
 
-- Zwycięzcy tego konkursu otrzymują tytuł „Elite Mapper: Aspirant” na forum oraz unikalną odznakę profilową Aspire.
+- Zwycięzcy tego konkursu otrzymują tytuł „Elite Mapper: Aspirant" na forum oraz unikalną odznakę profilową Aspire.
```

Comment:
```suggestion
- Zwycięzcy tego konkursu otrzymują tytuł "Elite Mapper: Aspirant" na forum oraz unikalną odznakę profilową Aspire.
```

you forgot this?

Reply:
no, https://discord.com/channels/188630481301012481/218677502141399041/1201938811702235166